api/src/main/java/org/hisp/dhis/android/dashboard/api/utils/DbUtils.java

```diff
@@ -94,14 +94,11 @@ public void run() {
                 continue;
             }
 
-            // if the last updated field in up to date model is after the same
```

Comment:
In general, all code has a sense for me but I have doubts with this removed code, can you explain to me because this code must be removed?